website/content/docs/commands/tls/ca.mdx

```diff
@@ -0,0 +1,45 @@
+---
+layout: docs
+page_title: 'Commands: TLS CA Create'
+---
+
+# Nomad TLS CA Create
+
+Command: `nomad tls ca create`
+
+This command create a self signed CA to be used for Nomad TLS setup.
+
+## Example
+
+Create CA:
+
+```shell-session
+$ nomad tls ca create
+==> Saved nomad-ca.pem
+==> Saved nomad-ca-key.pem
+```
+
+## Usage
+
+Usage: `nomad tls ca create [filename-prefix] [options]`
+
+#### Command Options
+
+- `-additional-name-constraint=<value>` - Add name constraints for the CA.
+  Results in rejecting certificates for other DNS than specified. Can be used
+  multiple times. Only used in combination with `-name-constraint`.
```

Comment:
Let's turn these into full sentences (and similar for the `-name-constraint` option):

```suggestion
- `-additional-name-constraint=<value>` - Add name constraints for the CA.
  The server will reject certificates for DNS names other than those specified. Can
  be used multiple times. This option can only used in combination with `-name-constraint`.
```